x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/TransformTask.java

```diff
@@ -240,9 +240,8 @@ void start(Long startingCheckpoint, ActionListener<StartTransformAction.Response
         logger.debug("[{}] start called with state [{}].", getTransformId(), getState());
         if (context.getTaskState() == TransformTaskState.FAILED) {
             listener.onFailure(
-                new ElasticsearchStatusException(
-                    TransformMessages.getMessage(CANNOT_START_FAILED_TRANSFORM, getTransformId(), context.getStateReason()),
-                    RestStatus.CONFLICT
```

Comment:
This wasn't even making it back to the user as a `409` right? Because it gets consumed in the `startTaskListener`?

Reply:
That's correct.
Primary reason it could not make it back to the user was that this is an asynchronous operation performed on a transform persistent task, **not** user-initiated API call.